index.md

```diff
@@ -101,6 +101,11 @@ r1(sin(1+z) + sin(1+w)) === nothing
 
 If you want to match a variable number of subexpressions at once, you will need a **segment variable**. `~~ys` in the following example is a segment variable:
 
+```julia:rewrite2.5
+@rule(+(~~xs) => ~~xs)(simplify(x + y + z))
```

Comment:
why is there a simplify here?

Reply:
My bad, I had thought `x + y + z` would create `(+)((+)(x, y), z)` but I was mistaken.